jwst/datamodels/tests/test_history.py

```diff
@@ -1,96 +0,0 @@
-import datetime
```

Comment:
Should we keep this file for now, since these tests aren't running yet in stdatamodels?

Reply:
We could. It's not testing anything in `jwst.datamodels` though.  We'll just have to make a note to port the tests over to `stdatamodels`.

Is there a reason these tests are disabled in `stdatadmodels`?  Is it just a lack of a `history` schema entry location?

Reply:
Asking the good questions!  I tried enabling them and they all passed.  I think they were failing before due to the model class test in the FITS hash code, which we fixed.  I'll open a PR over there to get those running.